### PR TITLE
Support `environment` configuration

### PR DESCRIPTION
**Pull request by Perdolique**
The existing plugin doesn't work with multi-environment configurations. To get the correct binding from `getPlatformProxy` in this case we need to specify `environment`.

### Example `wrangler.toml`

```toml
name = "woof"
compatibility_date = "2024-06-29"
pages_build_output_dir = "./dist"

[[env.production.d1_databases]]
binding = "DB"
database_name = "woof-prod"
database_id = "prod"

[[env.preview.d1_databases]]
binding = "DB"
database_name = "woof-stage"
database_id = "stage"
```

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -74,6 +75,7 @@ async function nitroModule(nitro: Nitro) {
       nitro.options.cloudflareDev?.shamefullyPatchR2Buckets,
     configPath,
     persistDir,
+    environment: nitro.options.cloudflareDev?.environmnet,
```
Comment (dario-piotrowicz) [inline src/index.ts:78]:
```suggestion
    environment: nitro.options.cloudflareDev?.environment,
```

File: src/index.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ declare module "nitropack" {
   interface NitroOptions {
     cloudflareDev?: {
       configPath?: string;
+      environmnet?: string;
```
Comment (dario-piotrowicz) [inline src/index.ts:16]:
```suggestion
      environment?: string;
```

**Review by dario-piotrowicz — APPROVED** [review]
Looks good to me :slightly_smiling_face:

**Review by pi0 — APPROVED** [review]
💯

**Pull request merged**